src/dbnode/persist/fs/seek_manager.go

```diff
@@ -111,19 +118,27 @@ type seekersAndBloom struct {
 	volume      int
 }
 
-// borrowableSeeker is just a seeker with an additional field for keeping track of whether or not it has been borrowed.
+// borrowableSeeker is just a seeker with an additional field for keeping
+// track of whether or not it has been borrowed. A seeker is essentially
+// a wrapper around file descriptors around a set of files, allowing for
+// interaction with them. We can ask a seeker for a specific time series, which
```

Comment:
"A seeker is essentially..." I think any clarification about what a seeker does should probably be a comment on the `ConcurrentDataFileSetSeeker` interface

Reply:
I move that part to the `ConcurrentDataFileSetSeeker`interface.